swagger/v2/api_spec.json

```diff
@@ -3259,35 +3051,20 @@
             "example": [
               {
                 "id": "db3a7848-7308-4879-942a-c4a70ced400a",
-                "type": "npq_application",
```

Comment:
I think this needs to stay as NPQ application as its under `"MultipleNPQApplicationsResponse`, same for attributes below?